custom_components/tuya_local/devices/confortotal_minisplit.yaml

```diff
@@ -0,0 +1,175 @@
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: auto
+              value: heat_cool
+            - dps_val: cold
+              value: cool
+            - dps_val: wind
+              value: fan_only
+            - dps_val: wet
+              value: dry
+            - dps_val: hot
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 16
+        max: 32
+      unit: C
+      mapping:
+        - step: 1
+    - id: 3
+      type: integer
+      name: current_temperature
+      unit: F
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      type: string
+      name: fan_mode
+      mapping:
+        - dps_val: strong
+          value: strong
+        - dps_val: auto
+          value: auto
+        - dps_val: low
+          value: low
+        - dps_val: middle
+          value: medium
+        - dps_val: high
+          value: high
+        - dps_val: mute
+          value: quiet
+    - id: 22
+      type: bitfield
+      name: fault_code
+    - id: 25
+      name: preset_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: sleep
+        - dps_val: false
+          value: comfort
+    - id: 31
+      type: string
+      name: vertical
+      hidden: true
+    - id: 33
+      type: boolean
+      name: swing_horiz
+    - id: 36
+      type: boolean
+      name: display
+secondary_entities:
+  - entity: switch
+    name: Ionizer
+    icon: "mdi:creation"
+    category: config
+    dps:
+      - id: 11
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Emergency heat
+    icon: "mdi:fire"
+    category: config
+    dps:
+      - id: 12
+        type: boolean
+        name: switch
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 22
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+  - entity: select
+    name: Vertical swing
+    icon: "mdi:angle-acute"
+    category: config
+    dps:
+      - id: 31
+        type: string
+        name: option
+        mapping:
+          - dps_val: "off"
+            value: "Off"
+          - dps_val: same
+            value: Swing
+          - dps_val: vane_1
+            value: Position 1
+          - dps_val: vane_2
+            value: Position 2
+          - dps_val: vane_3
+            value: Position 3
+          - dps_val: vane_4
+            value: Position 4
+          - dps_val: vane_5
+            value: Position 5
+  - entity: light
+    name: Display
+    icon: "mdi:television-ambient-light"
+    category: config
+    dps:
+      - id: 36
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Timer on
+    translation_key: timer
```

Comment:
Don't use translation_key here, as the name is non-standard.  For now, add `icon: timer` instead, and when translation parameters has been implemented it can be changed back to using translations.
But maybe "On timer" and "Off timer" would be better names.

Reply:
Updated `icon: timer` instead of `translation_key` and updated name.